src/common/WixDistribution.cs

```diff
@@ -34,6 +34,11 @@ internal static class WixDistribution
         public static string ShortProduct = "WiX Toolset";
 
         /// <summary>
+        /// Product name for the distribution.
+        /// </summary>
+        public static string Product = "Windows Installer XML Toolset";
```

Comment:
We're not called "Windows Installer XML Toolset" any more (we should fix places that use this old name). It's just "WiX Toolset" now.